irodsConnector/manager.py

```diff
@@ -197,8 +197,9 @@ def session(self) -> session.Session:
 
     @session.deleter
     def session(self):
-        del self._session
-        self._session = None
+        if self._session:
```

Comment:
This was removed when it was found no longer to be necessary (after removing a circular reference).  It must be determined why it suddenly is needed again.  The reason is important as may potentially cause other issues.